confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -18,65 +18,41 @@
 #
 
 ####################
-### thrift rpc configuration
+### Startup configuration
 ####################
 
+
 # could set ip or hostname
 # Datatype: String
-config_node_rpc_address=0.0.0.0
+rpc_address=0.0.0.0
 
-# used for communication between data node and config node
-# Datatype: int
-config_node_rpc_port=22277
 
-# used for communication between config node and config node
+# Used for cluster interior RPC communication
 # Datatype: int
-config_node_internal_port=22278
+rpc_port=22277
 
-# this feature is under development, set this as false before it is done.
-# Datatype: boolean
-# rpc_thrift_compression_enable=false
-
-# if true, a snappy based compression method will be called before sending data by the network
-# Datatype: boolean
-# this feature is under development, set this as false before it is done.
-# rpc_advanced_compression_enable=false
-
-# Datatype: int
-# rpc_max_concurrent_client_num=65535
-
-# thrift max frame size, 512MB by default
-# Datatype: int
-# thrift_max_frame_size=536870912
 
-# thrift init buffer size
+# Used for ConfigNodeGroup's ConsensusLayer interior communication
 # Datatype: int
-# thrift_init_buffer_size=1024
+consensus_port=22278
 
-# Thrift socket and connection timeout between raft nodes, in milliseconds.
-# Datatype: int
-# connection_timeout_ms=20000
 
-# selector thread (TAsyncClientManager) nums for async thread in a clientManager
-# Datatype: int
-# selector_thread_nums_of_client_manager=1
+# Used for connecting to the ConfigNodeGroup
+# Format: ip:port
+# where the ip should be consistent with the target ConfigNode's confignode_rpc_address,
+# and the port should be consistent with the target ConfigNode's confignode_rpc_port.
+# For the first ConfigNode to start, target_configNode points to its own ip:port.
+# For other ConfigNodes that are started or restarted, target_ConfigNode points to any running ConfigNode's ip:port.
+# Datatype: String
+target_confignode=0.0.0.0:22277
```

Comment:
why use this parameter?

Reply:
In order to simplify user operations. User can now just specify an active ConfigNode to start a node and join the cluster. To make the startup process more fault tolerant,  we can later extend this parameter to a list.